integreat_cms/summ_ai/utils.py

```diff
@@ -0,0 +1,106 @@
+# import os
+import logging
+import requests
+
+# from django.conf import settings
+from django.apps import apps
+from django.contrib import messages
+from django.utils.translation import ugettext as _
+
+logger = logging.getLogger(__name__)
+
+# pylint: disable=too-few-public-methods
+class SummAiAPI:
+    """
+    Summ AI API to get german posts in easy german language.
+    """
+
+    def __init__(self):
+        """
+        Initialize the Summ AI client
+        """
+        url = "https://backend.summ-ai.com/translate/"
+        self.translator = requests.get(url)
```

Comment:
The translator is typically a client object itself. If we don't have a fully featured api client for the API yet, we have to implement the request methods ourselves, but it does not make sense to make only one dummy request in the init method if the subsequent requests cannot build on it.